### PR TITLE
[15.0][MIG] purchase_requisition_operating_unit

### PR DESCRIPTION
**Pull request by Saran440**
- Add test script for cover all code
- Add depend on `stock_operating_unit`
- Change depend on `purchase_requisition` to `purchase_requisition_stock`

Depend on:
- [x] stock_operating_unit
- [x] purchase_requisition_stock
- [x] purchase_operating_unit - https://github.com/OCA/operating-unit/pull/443

### CODE REVIEW

File: purchase_requisition_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# © 2016 ForgeFlow S.L. (https://www.forgeflow.com)
+# © 2016 Serpent Consulting Services Pvt. Ltd.
```
Comment (AaronHForgeFlow) [inline purchase_requisition_operating_unit/__manifest__.py]:
```suggestion
# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
```
Can you change that in all headers please?

Comment (Saran440) [reply purchase_requisition_operating_unit/__manifest__.py]: @AaronHForgeFlow Fixed it all.

**Review by TheerayutEncoder — APPROVED** [review]
LGTM :+1:

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**